public/app/features/explore/NoDataSourceCallToAction.tsx

```diff
@@ -2,9 +2,16 @@ import React from 'react';
 import { css } from '@emotion/css';
 import { LinkButton, CallToActionCard, Icon, useTheme2 } from '@grafana/ui';
 
+// Services & Utils
+import { contextSrv } from 'app/core/core';
+// Types
+import { AccessControlAction } from 'app/types';
+
```

Comment:
Please remove all // Types and // Services & Utils separation comments, we don't use this anymore.
```suggestion
import { contextSrv } from 'app/core/core';
import { AccessControlAction } from 'app/types';

```